test/WebJobs.Script.Tests.Integration/Storage/AzureStorageProviderTests.cs

```diff
@@ -0,0 +1,234 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Storage.Blobs;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Options;
+using Xunit;
+
+namespace Microsoft.Azure.WebJobs.Script.Tests.Integration.Storage
+{
+    public class AzureStorageProviderTests
+    {
+        private const string StorageConnection = "AzureWebJobsStorage";
+
+        [Fact]
+        public async Task TestAzureStorageProvider_ConnectionInWebHostConfiguration()
+        {
+            var testData = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase)
+            {
+                { "ConnectionStrings:AzureWebJobsStorage", Environment.GetEnvironmentVariable(StorageConnection) },
```

Comment:
Instead of grabbing this straight from `Environment` -- use `TestHelpers.GetTestConfiguration()` and pull it from there. That combines env vars with an optional test settings file (which is what I use locally) -- so it's more flexible, and what other tests do.